conf/broker.conf

```diff
@@ -274,6 +274,24 @@ athenzDomainNames=
 # When this parameter is not empty, unauthenticated users perform as anonymousUserRole
 anonymousUserRole=
 
+### --- Token Authentication Provider --- ###
+
+## Simmetric key
+# Configure the secret key directly in this config file
+tokenSecretKey=
+# Read secret key from an environment variable
+tokenSecretKeyFromEnv=
+# Read secret key from a file
+tokenSecretKeyFromFile=
```

Comment:
Are there any case that we want to read tokenSecretKey from both Env and File?
How about using URL scheme?
```
tokenSecretKey=env:///FOO_BAR
tokenSecretKey=file:///path/for/file
```

Once we added support for "env:///" (not sure if we need 3 slashes) to `org.apache.pulsar.client.api.url.URL`, every places that use the class would support reading configuration from environment variable.

Reference: #672, #1212

Reply:
not really related, but Tim Berners Lee has said the // in http is a mistake. I guess file took the double slash to ape http, the third is to make the path absolute. Env should have at most 2 slashes, but none are needed.

Reply:
Also, it becomes a URI not a URL since it's not really pointing to a location

Reply:
Good point. I'll merge the 3 keys into a single one and make sure the same conf also applies to `client.conf` for consistency.